client/my-sites/site-settings/disconnect-site/down-flow.jsx

```diff
@@ -76,9 +76,9 @@ function DownFlow( { confirmHref, backHref, site, recordTracksEvent: tracks } )
 					<FormattedHeader
 						isSecondary
 						align="left"
-						headerText={ translate( 'Troubleshoot Jetpack.' ) }
+						headerText={ translate( 'Troubleshoot Jetpack' ) }
 						subHeaderText={ translate(
-							'If your site is working but you’re still seeing this error, let’s troubleshoot your Jetpack connection.'
+							'If your site is loading but you’re still seeing this error, this guide will help you troubleshoot the Jetpack connection.'
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.